packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 23.0.0
+
+* Relocates `lib` files to `src` directory.
```

Comment:
How about "Moves some files in `lib` that were not intended for direct client use to `lib/src`."